src/sonic-config-engine/tests/test_cfggen.py

```diff
@@ -52,13 +52,12 @@ def tearDown(self):
             pass
 
     def run_script(self, argument, check_stderr=False, verbose=False):
-        print('\n    Running sonic-cfggen ' + argument)
+        print('\n    Running sonic-cfggen ', argument)
```

Comment:
>, [](http://example.com/codeflow?start=43&length=1)

Does semgrep alert on this line?

Reply:
No, but `argument` variable is a list, cannot be concatenated with string in the print statement.

Reply:
Updated to `shlex.join`

Reply:
I notice it is not available in python 3.7. Then `+` is good enough in test code.